docs/rfcs/managed.md

```diff
@@ -0,0 +1,142 @@
+# Managed CRD
+
+**Status:** *Draft*
+
+**Author:** Antoine Toulme (@atoulme)
+
+**Date:** 2025-03-20
+
+## Objective
+
+Condense all best practices and deployment patterns of OpenTelemetry components into a single custom resource
```

Comment:
How does the proposal handle customizations done by the operator providers? For instance on OpenShift most likely there will be need to add a custom (OCP) related components.

Reply:
It should pick the best approach based on the environment. So on Openshift it would behave differently out of the box, installing those additional components.

Reply:
added something to that effect.